packages/vscode-extension/src/debug/launch.ts

```diff
@@ -28,3 +35,32 @@ export async function openHubWebClient(
 
   await VS_CODE_UI.openUrl(sideloadingUrl);
 }
+
+export async function openUrlInPrivateWindow(url: string): Promise<void> {
+  try {
+    await open(url, {
+      app: [
+        {
+          name: open.apps.chrome,
+          arguments: ["--incognito"],
+        },
+        {
+          name: open.apps.edge,
+          arguments: ["-inprivate"],
+        },
+        {
+          name: open.apps.firefox,
+          arguments: ["-private"],
+        },
+      ],
+    });
+  } catch {
+    const error = new UserError(
+      ExtensionSource,
+      ExtensionErrors.OpenBrowserFailed,
+      util.format(getDefaultString("teamstoolkit.localDebug.openBrowserFailed"), url),
+      util.format(localize("teamstoolkit.localDebug.openBrowserFailed"), url)
+    );
+    showError(error);
```

Comment:
Is telemetry track needed for this error?

Reply:
It think no need. Three reasons:
1. Opening bot framework protal is just a workaround since automatically adding Outlook channel (just like adding Teams channel) is not ready currently. When it is ready, this workaround will be retired.
2. This error is less likely to happen, only if there are no Chrome, Edge and Firefox in user's system.
3. This is not a blocking issue, even if error happens, users can open the link manually.